job-board-example/upload-indeed-listings.ts

```diff
@@ -1,13 +1,22 @@
 import * as fs from "fs";
-import * as readline from "readline";
-import * as path from "path";
 import { parse } from "csv-parse";
-import {
-  ChunkApi,
-  ChunkData,
-  Configuration,
-  CreateChunkData,
-} from "@devflowinc/trieve-js-ts-client";
+
+import dotev from "dotenv";
+
+dotev.config();
```

Comment:
```suggestion
import dotenv from "dotenv";

dotenv.config();
```

Reply:
My bad we actually don't need this because bun does it automatically

Reply:
Removed